git-hub

```diff
@@ -1147,13 +1147,13 @@ class IssueCmd (CmdGroup):
             if args.created_by_me and args.assigned_to_me:
                 issues = [
                         i for i in issues
-                        if filter(i, 'assignee')
-                        or filter(i, 'user')]
+                        if list(filter(i, 'assignee'))
+                        or list(filter(i, 'user'))]
```

Comment:
I think all the added `list()` here are not necessary. This is done by `2to3` because `filter()` returns a `list` in python2 but an iterator in python3, but I think we only use the results for iterating, so there should be no need to convert the iterator to a list.

Reply:
Not just unnecessary; it's outright wrong.
`filter` here is not the builtin; it's a custom function that returns `True` or `None`.

Reply:
Right. Missed that.